src/Native/pkg/runtime.native.System.IO.Compression/win10/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -8,9 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <!-- use Win10 temporarily as the distinguishing factor, in reality the distinction is app-container vs not. -->
     <File Include="$(WinNativePath)..\..\NetCoreForCoreCLR\native\clrcompression.dll">
-      <TargetPath>runtimes/win10-$(PackagePlatform)/native</TargetPath>
+      <TargetPath>runtimes/win10-$(PackagePlatform)/lib/netcore50</TargetPath>
```

Comment:
Hm, this seems correct according to #7436, but I'm just wondering if NuGet is going to think this is a "Compile" asset now? It's a ".dll" file under the lib/netcore50 folder, so I'm not sure what NuGet will do here, exactly.